.github/workflows/release.yml

```diff
@@ -20,14 +23,15 @@ jobs:
         with:
           go-version: 1.19.5
 
-      - name: Build kapp Script
-        run: |
-          ./hack/build-binaries.sh
-          shasum -a 256 ./kapp-* | tee checksums.txt
-          echo "# :open_file_folder: Files Checksum" | tee checksums-formatted.txt
-          echo \`\`\` | tee -a checksums-formatted.txt
-          cat checksums.txt | tee -a checksums-formatted.txt
-          echo \`\`\` | tee -a checksums-formatted.txt
+      - name: Run GoReleaser
+        # GoReleaser v4.2.0
```

Comment:
nit: This should also be removed I think?

Reply:
Since we are using the commit below, it is good to know this commit belongs to which tag.